modules/azure/aks/aks.tf

```diff
@@ -53,30 +53,22 @@ resource "azurerm_kubernetes_cluster" "this" {
   default_node_pool {
     name           = "default"
     vnet_subnet_id = data.azurerm_subnet.this.id
+    zones          = ["1", "2", "3"]
 
-    type                         = "VirtualMachineScaleSets"
-    zones                        = ["1", "2", "3"]
-    enable_auto_scaling          = false
-    only_critical_addons_enabled = true
-
-    orchestrator_version = var.aks_config.default_node_pool.orchestrator_version
-    node_count           = 1
+    orchestrator_version = var.aks_config.version
     # This is a bug in tflint
     # tflint-ignore: azurerm_kubernetes_cluster_default_node_pool_invalid_vm_size
-    vm_size = "Standard_D2as_v4"
-
-    node_labels = var.aks_config.default_node_pool.node_labels
+    vm_size                      = "Standard_D2ds_v5"
```

Comment:
Azure docs wants larger nodes but I do not think we need them if we are using node local dns as the load on coredns will be a lot lower.
https://docs.microsoft.com/en-us/azure/aks/use-system-pools